packages/block-library/src/quote/deprecated.js

```diff
@@ -109,7 +187,7 @@ const v2 = {
 			type: 'string',
 		},
 	},
-	migrate: migrateToQuoteV2,
+	migrate: compose( migrateToQuoteV2, migrateTextAlign ),
```

Comment:
Some small nitpicks.. You're using this and `migrateTextAlign( migrateToQuoteV2( attributes ) );`. Can we just use the second version and not load `compose`. I mean it's just a function call.

Reply:
Like this?
```
	migrate( attributes ) {
		return migrateTextAlign( migrateToQuoteV2( attributes ) );
	},
```

Reply:
Yes